tensorflow/python/kernel_tests/nn_ops/weighted_moments_test.py

```diff
@@ -0,0 +1,19 @@
+import tensorflow as tf
```

Comment:
We don't import TF within TF because that causes __all__ of TF to be imported even though we only need a subset of it for the test.